include/cuco/operator.hpp

```diff
@@ -290,7 +290,7 @@ struct find_tag {
  *
  * API Signature:
  * ```cpp
- * template <int32_t BlockSize,
+ * template <::int BlockSize,
```

Comment:
Why is the `::` needed?

Reply:
me just quoting @srinivasyadav18 "it's fully qualified". Ideally, this would be the right approach, but I’m not sure we should enforce it at this point.